browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -477,6 +436,17 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 					'command': 'exportpdf'
 				}
 			] : []);
+		} else if (docType === 'drawing') {
+			submenuOpts = [
+				{
+					'action': 'downloadas-odg',
+					'text': _('ODF Drawing (.odg)')
+				},
+				{
+					'action': 'downloadas-png',
```

Comment:
I noticed we don't have icon for download png

Reply:
Yes need to add this in notebookbar.css
```
[class*='as-png-submenu-icon'] {
	background: url('images/lc_insertgraphic.svg') center;
}
```